### DISCUSSION TITLE
Any chance to see tikz support implemented?

### DISCUSSION THREAD

**Discussion by boborbt** [marp-team/marp#266] [Ideas]
Tikz can be very useful to incorporate simple graphs in our presentation. Is there any chance to see it implemented?

In some other markdown-based tools I see it works, for instance using this syntax:

```
    ```latex {cmd=true hide=true}
    \documentclass{standalone}
    \usepackage{tikz}
    \begin{document}
    \begin{tikzpicture}
    % tikz commands here, e.g., 
    \draw (0,0) -- (1,1);
    \end{tikzpicture}
    \end{document}
```

**Comment by yhatt** [comment]
Marp Core's TeX support powered by [KaTeX](https://katex.org/) and [MathJax](https://www.mathjax.org/) is just only for math, and don't have any plans to support more of TeX packages unless these libraries decided to port the package.

Or you can try web based library like https://github.com/kisonecat/tikzjax with raw HTML within Markdown.
